src/xunit.runner.visualstudio/xunit.runner.visualstudio.nuspec

```diff
@@ -18,8 +18,15 @@
     <dependencies>
       <group targetFramework="net462">
         <dependency id="Microsoft.TestPlatform.ObjectModel" version="$MicrosoftTestPlatformObjectModelVersion$" />
+        <dependency id="Microsoft.Testing.Extensions.Telemetry" version="$MicrosoftTestingPlatformVersion$" />
+		<dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="$MicrosoftTestingPlatformVersion$" />
```

Comment:
Not sure why the indent is all messed up, I don't see that locally:

![Uploading image.png…]()